sherpa/astro/ui/tests/test_astro_ui_unit.py

```diff
@@ -431,17 +431,12 @@ def test_save_data_data1d_no_clobber(tmp_path):
 def test_save_data_datapha_no_clobber(tmp_path):
     """save_data: does clobber=False work? DataPHA"""
 
-    # This import requires an I/O backend hence it is done here
-    #
-    from sherpa.astro.io.meta import Meta
-
     ui.load_arrays(1, [1, 2, 3], [5, 4, 3], ui.DataPHA)
 
     # The code requires DataPHA to have a "valid" header so
     # fake one. Ideally we would not require it.
     #
-    hdr = Meta()
-    ui.get_data().header = hdr
+    ui.get_data().header = {}
```

Comment:
I wonder if we should just make `DataPHA` set the `header` attribute duriing initialization?

Reply:
I think that's a good idea! Before I put that in though, I want to make all the current tests pass. There are some real failures that I need to understand because it indicates that there is some subtle difference between the dict and the old implementation. I suspect that it's something that I can fix, but I need to understand what's going on.